solc/CommandLineInterface.cpp

```diff
@@ -989,6 +989,13 @@ bool CommandLineInterface::processInput()
 		serr() << "Exception during compilation: " << boost::diagnostic_information(_exception) << endl;
 		return false;
 	}
+	catch (std::exception const& _e)
+	{
+		serr() << "Unknown exception during compilation" << (
+			_e.what() ? ": " + string(_e.what()) : "."
+		) << "." << endl;
```

Comment:
this will print two dots when `_e.what()` is null

Reply:
Should be fixed now.